packages/manager/modules/pci/src/projects/project/public-gateways/delete/delete.routing.js

```diff
@@ -0,0 +1,17 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('pci.projects.project.public-gateways.delete', {
+    url: '/delete',
+    params: { gateway: null },
+    views: {
+      modal: {
+        component: 'pciProjectPublicGatewaysDelete',
+      },
+    },
+    resolve: {
+      goBack: /* @ngInject */ (goToPublicGateway) => goToPublicGateway,
+      gateway: /* @ngInject */ ($transition$) => $transition$.params().gateway,
+      breadcrumb: () => null,
+    },
+    layout: 'modal',
```

Comment:
**issue:** default page tracking must be blocked since page tracking is done in the controller `$onInit`
```suggestion
    layout: 'modal',
    atInternet: {
      ignore: true,
    },
```

Reply:
updated as suggested.